kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -1078,7 +1069,7 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
         val actions = inMemoryIterable(stageData(txn1State, Map.empty, dataBatches2))
         expData ++= dataBatches2.flatMap(_.toTestRows)
 
-        val txn1Result = txn1.commit(engine, actions)
+        val txn1Result = commitTransaction(txn1, engine, actions)
```

Comment:
Do you think we can just one-off validate the crcInfo has the right numAddFiles after this commit? I feel like there should be some amt of coverage to make sure we write the correct crcInfo in the case of a conflicting txn